Classes/Parser/NodeFactory.php

```diff
@@ -106,12 +106,12 @@ public function buildClassNode($classObject, $skipStatements = false)
     }
 
     /**
-     * @param \EBT\ExtensionBuilder\Domain\Model\File $fileObject
+     * @param File $fileObject
      * @return array
      */
     public function getFileStatements(File $fileObject)
     {
-        $stmts = [];
+        $stmts = $this->getContainerStatements($fileObject);
```

Comment:
This resulted in double generated class code in each file when I tested it (and that's the error in CI Unit Tests).
I think we have to add a flag to the FileObject whether or not to add the declare strict line...